open-metadata-resources/open-metadata-archives/design-model-archives/src/main/java/org/odpi/openmetadata/archiveutilities/designmodels/owlcanonicalglossarymodel/properties/ModelElement.java

```diff
@@ -0,0 +1,226 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.archiveutilities.designmodels.owlcanonicalglossarymodel.properties;
+
+import java.util.*;
+
+/**
+ * Attribute describes an attribute in the model
+ */
+public class ModelElement
```

Comment:
Javadoc missing on methods - this is a generic comment as occurs in other files

Reply:
fixed